### DISCUSSION TITLE
Distributors and parts in USA

### DISCUSSION THREAD

**Discussion by A-I-Ivanov** [bitcraze/discussions#1589] [Q&A] [answered]
I bought a drone a few years ago a to play around with, and now need some spare parts. I tried to buy parts on the webpage but shipping is ~95 USD for ~25USD worth of spare parts, which I cannot justify. I tried going through the listed USA retailers, but some of the webpages don't exist (DIIGIIT), and others simply don't have parts. How do I get spare parts in the USA without exorbitant shipping fees?

**Comment by Bananazooka** [comment]
Hi, 
I understand that the shipping costs can be high for just spare parts. One option you could use is to have them shipped with the normal post. It may take longer and there is no tracking, but shipping is only 5$ then.

**Reply by A-I-Ivanov** [reply]
I don't mind slow shipping and no tracking, but this didn't seem like there was an option for this on the website.  Only FedEx and FedEx priority.  Do I have to make some sort of special request?

**Reply by Bananazooka** [reply, marked as answer]
Did you have batteries in the shipment? If the package is too big, or if it contains batteries, unfortunately we can't use the Post and it has to be FedEx.

**Reply by A-I-Ivanov** [reply]
Ok that works. Thank you for the information.  It was not at all clear that this was the issue.
